container.nix

```diff
@@ -431,6 +455,7 @@ let
 
   serviceConfigDefault = {
     Restart = "always";
+    Environment = "PATH=/run/wrappers/bin:/run/current-system/sw/bin";
```

Comment:
This is needed because podman requires the `newuidmap` binary to exist in the PATH, otherwise you get the error:

```
Error: command required for rootless mode with multiple IDs: exec: "newuidmap": executable file not found in $PATH
```

If I add `PATH=${pkgs.shadow}/bin` it doesn't work becuase the `newuidmap` binary directly from the package isn't suid:

```
Error: cannot set up namespace using "/nix/store/1cb448wfsb9a6igzg7pmr18387886w9a-shadow-4.14.6/bin/newuidmap": should have setuid or have filecaps setuid: exit status 1
```

Hence relying on `/run/wrappers/bin`

I wanted to add this only for rootless containers, but since the `container.nix` config exports only the compiled systemd unit text, it's not really possible to munge that from within `home-manager-module.nix`

One thing I can do is drop `/run/current-system/sw/bin` as that isn't needed. I'll also add a comment.

Reply:
Thanks for the context. That makes sense.

I think it's fine to leave it as is for simplicity, as it doesn't hurt for rootful containers.